provisioning/playbook.yml

```diff
@@ -26,10 +28,11 @@
     - { role: geerlingguy.apache, when: drupalvm_webserver == 'apache' }
     - { role: geerlingguy.apache-php-fpm, when: drupalvm_webserver == 'apache' }
     - { role: geerlingguy.nginx, when: drupalvm_webserver == 'nginx' }
-    - geerlingguy.mysql
     - geerlingguy.php
     - geerlingguy.php-pecl
-    - geerlingguy.php-mysql
+    - { role: geerlingguy.mysql, when: drupalvm_database == 'mysql' }
+    - { role: geerlingguy.php-mysql, when: drupalvm_database == 'mysql' }
+    - { role: ANXS.postgresql, when: drupalvm_database == 'pgsql' }
```

Comment:
I considered that too but then opted for the same setup we use with `drupalvm_webserver`. Maybe there are a few more people interested in using multiple database engines than multiple webservers though.